clients/teku-bn/teku_bn.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Immediately abort the script on any error encountered
+set -e
+
+if [ ! -f "/hive/input/genesis.ssz" ]; then
+    if [ -z "$HIVE_ETH2_ETH1_RPC_ADDRS" ]; then
+      echo "genesis.ssz file is missing, and no Eth1 RPC addr was provided for building genesis from scratch."
+      exit 1
+    fi
+fi
+
+mkdir -p /data/testnet_setup
+
+cp /hive/input/genesis.ssz /data/testnet_setup/genesis.ssz
+cp /hive/input/config.yaml /data/testnet_setup
+
+mkdir -p /data/teku
+
+LOG=INFO
+case "$HIVE_LOGLEVEL" in
+    0|1) LOG=ERROR ;;
+    2)   LOG=WARN  ;;
+    3)   LOG=INFO  ;;
+    4)   LOG=DEBUG ;;
+    5)   LOG=TRACE ;;
+esac
+
+echo "bootnodes: ${HIVE_ETH2_BOOTNODE_ENRS}"
+
+CONTAINER_IP=`hostname -i | awk '{print $1;}'`
+eth1_option=$([[ "$HIVE_ETH2_ETH1_RPC_ADDRS" == "" ]] && echo "" || echo "--eth1-endpoints=$HIVE_ETH2_ETH1_RPC_ADDRS")
+metrics_option=$([[ "$HIVE_ETH2_METRICS_PORT" == "" ]] && echo "" || echo "--metrics-enabled=true --metrics-interface=0.0.0.0 --metrics-port=$HIVE_ETH2_METRICS_PORT --metrics-host-allowlist=*")
+if [ "$HIVE_ETH2_MERGE_ENABLED" != "" ]; then
+    echo -n "0x7365637265747365637265747365637265747365637265747365637265747365" > /jwtsecret
+    merge_option="--ee-endpoint=$HIVE_ETH2_ETH1_ENGINE_RPC_ADDRS --ee-jwt-secret-file=/jwtsecret"
+fi
+
+/opt/teku/bin/teku \
+    --network=/data/testnet_setup/config.yaml \
+    --data-path=/data/teku \
+    --initial-state=/data/testnet_setup/genesis.ssz \
+    --eth1-deposit-contract-address="${HIVE_ETH2_CONFIG_DEPOSIT_CONTRACT_ADDRESS:-0x1111111111111111111111111111111111111111}" \
+    --logging="$LOG" \
```

Comment:
Otherwise it will go to log file only which is not collected
```suggestion
    --log-destination console \
    --logging="$LOG" \
```